### ISSUE TITLE
Contrast theme no longer meets minimum color contrast ratio requirements

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#7896]
[Updated] Solved in v3.0.0-beta.7 for Form Library and Form Builder
***
**Form Library Demo, v3**
[View Plunker (Contrast Light)](https://plnkr.co/edit/49D2vbjFuMKFkUvZ)

To reproduce the issue, run the demo and scan it with axe DevTools (Chrome). The [Elements must meet minimum color contrast ratio thresholds](https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=axeAPI) accessibility error occurs with the Contrast theme applied.

**Form Builder Demo, v2 and v3**

Go to https://surveyjs.io/survey-creator/examples/free-nps-survey-template/, select the **Contrast** theme.
Run Axe Check

<img width="1920" height="912" alt="Image" src="https://github.com/user-attachments/assets/24a8fa1c-52d7-4645-818e-2e2d0df74ebc" />

The following elements are problematic:
* sq_2076_ariaTitle

<img width="1920" height="912" alt="Image" src="https://github.com/user-attachments/assets/d5d54be4-d41b-408e-aac6-66bd95e3ddf3" />
 
* sq_2079_ariaTitle 

<img width="1920" height="912" alt="Image" src="https://github.com/user-attachments/assets/f1a91b72-155a-4e57-b3a3-a60d7916c6e2" />

**Comment by nisanord** [comment]
Fixed in recent updates for both Contrast and Monochrome themes. We need to retest our live-demos with AXE tool later.

**Comment by nisanord** [comment]
values for v2:

fg-basic-secondary texts & icons (average gray, such as labels): 75% opacity from fg-basic-primary (black) = rgba(0, 0, 0, 0.75)
fg-basic-tertiary texts & icons (light gray, such as hint buttons): 55% opacity from fg-basic-primary (black) = rgba(0, 0, 0, 0.55)

!!! use v2 analogue of v3 fg-basic-primary color instead of pure black — I guess it may be sjs-layer-3-foreground-100
